custom_components/elegoo_printer/elegoo_sdcp/models/status.py

```
@@ -1,11 +1,30 @@
 """Models for the Elegoo printer."""
 
 import json
-from typing import Any
+from typing import Any, List
 
 from .enums import ElegooMachineStatus, ElegooPrintError, ElegooPrintStatus
 
 
+class CurrentFanSpeed:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)